docs/guides/extending-warp-route.mdx

```diff
@@ -0,0 +1,178 @@
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+import PrerequisitesPartial from '/docs/partials/warp-routes/_prerequisites-config-symbol.mdx'
+import WarpReadSymbolChainPartial from '/docs/partials/warp-routes/commands/_warp-read-symbol-chain.mdx'
+import WarpApplySymbolConfigDefaultPartial from '/docs/partials/warp-routes/commands/_warp-apply-symbol-config-default.mdx'
+import WarpApplySymbolConfigDefaultStrategyPartial from '/docs/partials/warp-routes/commands/_warp-apply-symbol-config-default-strategy.mdx'
+import WarpReadSymbolSelectPartial from '/docs/partials/warp-routes/_warp-read-symbol-selection.mdx'
+
+# How to extend an existing Warp Route
+
+# Using the Hyperlane CLI
+
+After deploying your warp route, you may want to extend it to a new chain. This happens in 2 steps: new contract get deployed to the new chain, and all existing routers get enrolled with the new one.
+
+You can extend an existing warp route using the [Hyperlane CLI](https://docs.hyperlane.xyz/docs/reference/cli).
+<PrerequisitesPartial />
+:::warning
+Gnosis Safe Users: The private key that owns the warp route must either be a Safe signer or [delegate](https://docs.safe.global/core-api/transaction-service-guides/delegates) to be able to propose update transactions.
```

Comment:
Is this warning unique to extending warp routes or for all safe-based CLI txs?

Reply:
Right now this only applies to extending warp routes. In the future, i expect it to apply to others